### PR TITLE
Root should always have read access

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: docker/qgisserver/geomapfish_qgisserver/accesscontrol.py [hunk 1]
```diff
@@ -530,13 +530,16 @@ def layerPermissions(self, layer):  # NOQA
 
             session = self.DBSession()
             try:
+                roles = self.get_roles(session)
+
+                if roles == "ROOT":
```
Comment (jwkaltz) [inline docker/qgisserver/geomapfish_qgisserver/accesscontrol.py:535]: Is roles not a list (of roles)?

Comment (sbrunner) [reply docker/qgisserver/geomapfish_qgisserver/accesscontrol.py:535]: Not if we are root ...

**Review by arnaud-morvan — COMMENTED** [review]
Is it not already the case with : https://github.com/camptocamp/c2cgeoportal/blob/master/docker/qgisserver/geomapfish_qgisserver/accesscontrol.py#L402

**Comment by sbrunner** [comment]
@arnaud-morvan As I understand no because it's done here:
https://github.com/camptocamp/c2cgeoportal/blob/0f4a8793ba092d3cd846261b65bbe5525a0ab83e/docker/qgisserver/geomapfish_qgisserver/accesscontrol.py#L545-L550
And herer https://github.com/camptocamp/c2cgeoportal/blob/0f4a8793ba092d3cd846261b65bbe5525a0ab83e/docker/qgisserver/geomapfish_qgisserver/accesscontrol.py#L540-L541
We return if the layer is not in the gmf layers...

Do you agree or I miss something?

**Review by arnaud-morvan — APPROVED** [review]
Ok, I missed this return statement.

**Pull request merged**